app/Models/User.php

```diff
@@ -1075,6 +1075,43 @@ public function isModerator()
         return $this->isGMT() || $this->isNAT();
     }
 
+    /**
+     * Determine if the user has a "bad standing" based on recent infringements.
+     *
+     * @param User|null $viewer The user viewing the profile.
+     * @return bool
+     */
+    public function hasBadStanding(?User $viewer = null): bool
```

Comment:
change the name to `inBadStanding` to be more in line with wording on the website